Dependency - Flare Flutter version needs to be upgraded

When compiling the project, it was observed that:

> Compiler message:
../Flutter%20SDK/flutter/.pub-cache/hosted/pub.dartlang.org/flare_flutter-1.5.7/lib/flare.dart:601:14: Error: The method 'FlutterActor.makeShapeNode' has fewer positional arguments than those of overridden method 'Actor.makeShapeNode'.
ActorShape makeShapeNode() {
^
../Flutter%20SDK/flutter/.pub-cache/hosted/pub.dartlang.org/flare_dart-1.4.8/lib/actor.dart:68:14: Context: This is the overridden method ('makeShapeNode').
ActorShape makeShapeNode(ActorShape source) {
^
Compiler failed on C:\Users\Tomer\Documents\mobile-app\lib\main.dart

FAILURE: Build failed with an exception.

Where:
Script 'C:\Users\Tomer\Documents\Flutter SDK\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 765
What went wrong:
Execution failed for task ':app:compileFlutterBuildDebugX86'.
Process 'command 'C:\Users\Tomer\Documents\Flutter SDK\flutter\bin\flutter.bat'' finished with non-zero exit value 1

The current flare flutter version in the project is 1.5.7.

To mediate this issue, it is required to upgrade to the latest flare flutter [version](https://pub.dev/packages/flare_flutter/versions) - 1.5.15 

This issue originated from #22

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -35,7 +35,7 @@ dependencies:
   intl_translation: ^0.17.7
 
   # UI
-  flare_flutter: 1.5.7
+  flare_flutter: 1.5.15
 
 
 

